deployment/src/main/java/io/quarkiverse/renarde/deployment/RenardeProcessor.java

```diff
@@ -216,8 +216,11 @@ void setupSecurity(LaunchModeBuildItem launchMode, Capabilities capabilities,
             // those are the better defaults
             defineUnlessPresent("mp.jwt.token.header", "Cookie", config, runtimeConfigurationBuildItem);
             defineUnlessPresent("mp.jwt.token.cookie", "QuarkusUser", config, runtimeConfigurationBuildItem);
+            defineUnlessPresent("renarde.redirect-location", "quarkus-redirect-location", config,
```

Comment:
I don't think this is the right way to define a new config property. This is only useful to override properties defined in other extensions. See https://quarkus.io/guides/writing-extensions#configuration for how we can define configuration.

Reply:
Also, the name should be in line with the other redirect cookie config that it pretty much copies: https://quarkus.io/guides/all-config#quarkus-vertx-http_quarkus.http.auth.form.location-cookie so perhaps `quarkus.renarde.auth.location-cookie`?